index.html

```diff
@@ -1,47 +1,83 @@
 <!doctype html>
 <html lang="en">
-  <head>
+
+<head>
   <meta charset="utf-8">
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
-  <link rel="stylesheet"  type="text/css" href="application.css">
-  </head>
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
 <body>
   <h1 class="title-tech">Front-End Tools</h1>
   <div class="content">
-  <article class="tech">
-  <img class="tech-logo" src="assets/react_icon.svg" />
-  <span class="tech-title">ReactJS</span>
-  <span class="release">Release 2013</span>
-  <p class="text">React is a declarative, component-based JavaScript Library for building user interfaces.</p>
-  </article>
-  <article class="tech">
-  <img class="tech-logo" src="assets/angular_icon.svg" />
-  <span class="tech-title">AngularJS</span>
-  <span class="release">Release: 2010</span>
-  <p class="text">AngularJS (commonly reffered to as AngularJS 1.X) is a JavaScript framework for building SPAs with an MVC
-    architecture.</p>
-  </article>
-  <article class="tech">
-  <img class="tech-logo" src="assets/vue_icon.svg" />
-  <span class="tech-title">VueJS</span>
-  <span class="release">Release: 2014</span>
-  <p class="text">Vue.js (pronounced /vjuː/, like view) is an open-source JavaScript framework for building user interfaces.</p>
-  </article>
-  <article class="tech">
-  <img class="tech-logo" src="assets/elm_icon.svg" />
-  <span class="tech-title">Elm</span>
-  <span class="release">Release: 2016</span>
-  <p class="text">Elm is a domain-specific programming language for declaratively creating web browser-based graphical user
-    interfaces.</p>
-  </article>
-  <article class="tech">
-  <img class="tech-logo" src="assets/polymer_icon.svg" />
-  <span class="tech-title">Polymer</span>
-  <span class="release">Release: 2015</span>
-  <p class="text">Polymer is an open-source JavaScript library for building web applications using Web Components.</p>
-  </article>
+    <article class="tech">
+      <img class="tech-logo" src="assets/react_icon.svg" />
+      <span class="tech-title">ReactJS</span>
+      <span class="release">Release 2013</span>
+      <p class="text">React is a declarative, component-based JavaScript Library for building user interfaces.</p>
+    </article>
+    <article class="tech">
+      <img class="tech-logo" src="assets/angular_icon.svg" />
+      <span class="tech-title">AngularJS</span>
+      <span class="release">Release: 2010</span>
+      <p class="text">AngularJS (commonly reffered to as AngularJS 1.X) is a JavaScript framework for building SPAs
+        with an MVC
```

Comment:
Why is this line so short?

Should be something like this. Repeat in the rest of strange `<p>`s

```
<p class="text">
  AngularJS (commonly reffered to as AngularJS 1.X) is a JavaScript framework for building SPAs
  with an MVC architecture.
</p>
```